blog/2022-09-8-office-hours.mdx

```diff
@@ -44,7 +44,7 @@ that  when a contributor adds something to the examples, or to the open source s
 
 You can jump in here, the [KubeSlice Roadmap](https://github.com/orgs/kubeslice/projects/4/views/1), and feel free to take up any issue and again you can reach out on the  #kubeslice channel on the [Kubernetes slack](https://slack.k8s.io/) where we hang out, if you have questions about something or if you need any guidance and support. 
 Additionally, the Office Hours are intended 
-to be as interactive as possible and we’d love you to join us on our next one to be held on 8th September 2022. 
+to be as interactive as possible and we’d love you to join us on our next one to be held on 22nd September 2022. 
 To receive an invite for the next one in your inbox & stay updated with the latest goings-on in the project, [join our Google Group](https://groups.google.com/g/kubeslice-dev).
```

Comment:
invite -> invitation , looks better here

Reply:
I disagree because you do not send out meeting invitations, you send out meeting invites.